### PR TITLE
Create intellect-newgen-style.csl

### PR DESCRIPTION
**Pull request by amyrolla**
First CSL file for Intellect Books (sorry I placed this in dependent first by mistake)

### CODE REVIEW

File: intellect-newgen-style.csl [hunk 1]
```diff
@@ -0,0 +1,560 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="never" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Intellect Newgen Style</title>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:5]:
```suggestion
    <title>Intellect Newgen Books</title>
```

File: intellect-newgen-style.csl [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+    <id>http://www.zotero.org/styles/intellect-newgen-style</id>
+    <link href="http://www.zotero.org/styles/intellect-newgen-style" rel="self"/>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <summary>Style for Intellect Newgen Style</summary>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:10]:
```suggestion
    <summary>Style for Intellect Newgen Books</summary>
```

File: intellect-newgen-style.csl [hunk 3]
```diff
@@ -0,0 +1,560 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="never" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Intellect Newgen Style</title>
+    <id>http://www.zotero.org/styles/intellect-newgen-style</id>
+    <link href="http://www.zotero.org/styles/intellect-newgen-style" rel="self"/>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:7]: Add "template" and "documentation" links please

File: intellect-newgen-style.csl [hunk 4]
```diff
@@ -0,0 +1,560 @@
...
+      </else>
+    </choose>
+  </macro>
+  <macro name="year">
+    <choose>
+      <if variable="original-date">
+        <date variable="original-date">
+          <date-part name="year"/>
+        </date>
+      </if>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:30]: Should the publication year (`issued`) also be included when `original-date` is present?

File: intellect-newgen-style.csl [hunk 5]
```diff
@@ -0,0 +1,560 @@
...
+      <if variable="author editor translator" match="any">
+        <choose>
+          <if variable="container-title" match="none" type="thesis webpage speech musical_score legislation legal_case interview">
+            <text variable="title" font-style="italic" text-case="title"/>
+            <choose>
+              <if type="broadcast" match="any">
+                <group>
+                  <text value="DVD" prefix=", " suffix=", "/>
+                  <text value="extended" suffix=" "/>
+                  <label variable="edition" form="short"/>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:120]: What exactly is this for? In general, hardcoding these words seems inappropriate (what if the citation is not to a DVD?). I would suggest instead printing the `genre` variable and expecting this information to be in the item data.

File: intellect-newgen-style.csl [hunk 6]
```diff
@@ -0,0 +1,560 @@
...
+                </group>
+              </if>
+            </choose>
+            <choose>
+              <if type="graphic" match="any">
+                <text value="installation" prefix=", "/>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:126]: Art exhibits should be cited using type `event`, rather than `graphic`.

File: intellect-newgen-style.csl [hunk 7]
```diff
@@ -0,0 +1,560 @@
...
+                  <text variable="publisher" prefix=", "/>
+                  <text variable="publisher-place"/>
+                </group>
+              </if>
+            </choose>
+            <choose>
+              <if type="musical_score" match="any">
+                <group>
+                  <text term="editor" form="short" prefix=", ("/>
+                  <text value=" L. Bernstein), "/>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:137]: What is this for?

File: intellect-newgen-style.csl [hunk 8]
```diff
@@ -0,0 +1,560 @@
...
+                  <text value="score"/>
+                </group>
+              </if>
+            </choose>
+            <choose>
+              <if type="pamphlet" match="any">
+                <text value="exhibition catalogue" prefix=", "/>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:144]: This should not be hard-coded. The appropriate item type for an exhibition catalogue is `book`.

File: intellect-newgen-style.csl [hunk 9]
```diff
@@ -0,0 +1,560 @@
...
+              </if>
+            </choose>
+          </if>
+          <else>
+            <choose>
+              <if type="entry-dictionary legislation legal_case interview report" match="none">
+                <text variable="title" quotes="true"/>
+                <choose>
+                  <if type="review" match="any">
+                    <text value="review" prefix=", "/>
```
Comment (bwiernik) [inline intellect-newgen-style.csl:154]: Use `term="review"`

**Review by bwiernik — CHANGES_REQUESTED** [review]
Thanks for the submission!

A few notes to start:

1. Can you please add a "documentation" link to the style guidelines online?
2. Did you start from scratch or by modifying another style? Can you please add a "template" link for the style that you started with to edit?
3. Please don't hard-code in format/type data. Expect that to be in the `medium` or `genre` variables.
4. There are a bunch of apparent errors with author names included in the style. Please double check all of that.
5. You have a bunch of prefixes and suffixes with things like `, `. These should be groups with delimiters instead.

**Comment by POBrien333** [comment]
See also this open PR for this style: https://github.com/citation-style-language/styles/pull/5918

**Comment by bwiernik** [comment]
@amyrolla Can you please update your style per my reviews above? Please edit the style in the PR. Do not open a new PR

**Comment by amyrolla** [comment]
Apologies! I'm getting the style updated with our developers and will update the PR asap

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by adam3smith** [comment]
Closing due to inactivity -- we're happy to re-open this if you find time to come back to it: simply post a short message here.

**Pull request closed without merging**